packages/wrangler/src/config/validation.ts

```diff
@@ -220,7 +222,25 @@ function normalizeAndValidateBuild(
     );
   }
 
-  return { command, cwd, watch_dir, deprecatedUpload };
+  return {
+    command,
+    watch_dir:
+      // - `watch_dir` only matters when `command` is defined, so we apply
+      // a default only when `command` is defined
+      // - `configPath` will always be defined since `build` can only
+      // be configured in `wrangler.toml`, but who knows, that may
+      // change in the future, so we do a check anyway
+      command
+        ? configPath
+          ? path.relative(
+              process.cwd(),
```

Comment:
Generally I think we need to be making these paths relative to the wrangler.toml. The reason is that otherwise the following would have different results:

```bash
wrangler -c path/to/wrangler.toml ...
cd path/to
wrangler -c wrangler.toml
```

Reply:
sorry, I don't understand. what would be the difference with these two? they would both resolve a watch_dir config to the same path, unless I'm missing something

Reply:
it does resolve against wrangler.toml's path in the next line

Reply:
OK, so I see my confusion now. You are saying that the `configPath` is always, already, relative to the `cwd()` (or it is absolute).

In that case there is no need for the `relative(process.cwd(), ...)` bit then, right?

Reply:
Just a personal preference, I don't like seeing absolute paths if and when I log this during development. Is that a problem?

Reply:
This is exactly how we normalised paths for wasm_modules and text_blobs too

Reply:
OK - it just really confused me, because if it is possible that `configPath` is not relative to `src` then this would break.
Happy to go with this for the timebeing.

Reply:
Yeah of the wrangler.Tom isn't adjacent to a src directory then this won't work. Honestly I think there shouldn't have been a default for that field at all (much like upload.dir)